### ISSUE TITLE
[Minor Gripe] Notes flashes white when loading

### ISSUE THREAD

**Issue by tracyboehrer** [standardnotes/forum#963]
On both my Windows and Linux clients, with a dark theme set, clicking on notes will flash back to white while it's loading.  Interestingly enough, it doesn't do this on mobile (Android).

**Comment by JaspalSuri** [comment]
That's pretty strange. What theme are you using and does reinstalling it help? I can't reproduce this with the Futura or Midnight themes.

**Comment by 17prime** [comment]
Happens here too (Linux desktop or webapp), but only when switching between different notes that are using different editors (in my case, switching from the plain editor to the indent-editor) ... perhaps in part due to the delay of checking for a component update every single time a component (like an editor) is switched (even if it just checked five seconds ago).

**Comment by baptiste-grob** [comment]
Yes this is a known issue. I'm not quite sure how to deal with it yet but it's annoying to me too and I want to address it at some point :)

**Comment by tacohajo** [comment]
Same here...
Android flashes bright white 2 times while starting up... then goes darkmode (what I use)
Linux client flashes 1 time bright white before going dark mode....
Web app starts dark and stays dark on startup (both chrome and FF)
